### PR TITLE
Fix vuln test

### PR DESCRIPTION
**Pull request by janisz**
## Description

Looks like there is no property `tewakesortedVulnsdArray`.
This PR fixes this and also add types and use Set instead of ordered list.

### CODE REVIEW

File: qa-tests-backend/src/test/groovy/VulnMgmtSACTest.groovy [hunk 1]
```diff
@@ -240,9 +240,9 @@ class VulnMgmtSACTest extends BaseSpecification {
     }
 
     // GraphQL does not provide ordering guarantees, so to compare the results
-    // of two GraphQL queries we extract just the CVE names and sort them.
-    def extractCVEsAndSort(queryCallResult) {
-        return queryCallResult.results*.cve.sort()
+    // of two GraphQL queries we extract just the CVE names as a Set.
+    Set<String> extractCVEs(queryCallResult) {
+        return (queryCallResult.results*.cve.toSet() + UUID.randomUUID().toString()) as Set<String>
```
Comment (rhybrillou) [inline qa-tests-backend/src/test/groovy/VulnMgmtSACTest.groovy:245]: I suppose the extra UUID.randomUUID().toString() is there in order to have a failing test and validate the error reporting code.

Comment (janisz) [reply qa-tests-backend/src/test/groovy/VulnMgmtSACTest.groovy:245]: Correct, I'll revert it once I see it's working correctly

**Comment by janisz** [comment]
@rhybrillou Output is now cleaner and missing property is fixed.

**Review by rhybrillou — APPROVED** [review]

**Pull request merged**